PyViCare/PyViCareHeatingDevice.py

```diff
@@ -230,6 +230,10 @@ def getSolarPowerProduction(self):
     def getSolarPowerProductionUnit(self):
         return self.service.getProperty("heating.solar.power.production")["properties"]["day"]["unit"]
 
+    @handleNotSupported
+    def getSolarPowerProductionCumulativeProduced(self):
+        return self.service.getProperty("heating.solar.power.cumulativeProduced")["properties"]["value"]["value"]
```

Comment:
Please verify if this is excluded in

https://github.com/openviess/PyViCare/blob/master/tests/test_TestForMissingProperties.py